iso15118/secc/states/iso15118_2_states.py

```diff
@@ -857,24 +904,57 @@ async def process_message(
             # TODO Either an MO Root certificate or a V2G Root certificate
             #      could be used to verify, need to be flexible with regards
             #      to the PKI that is used.
-            verify_certs(leaf_cert, sub_ca_certs, load_cert(CertPath.MO_ROOT_DER))
-
-            # TODO Check if EMAID has correct syntax
+            root_cert_path = self._mobility_operator_root_cert_path()
+            verify_certs(leaf_cert, sub_ca_certs, load_cert(root_cert_path))
```

Comment:
we dont have a guarantee that the MO root will be available in the station. So, we may actually call a evse_controller to get a True/False result for the question "verify contract certificate chain?"

Reply:
https://github.com/SwitchEV/iso15118/issues/94